### PR TITLE
switching back to normal styling and adding the history page

### PR DESCRIPTION
**Pull request by narendasan**
Switches the Logos and the colors back to our normal branding. Adds a history page, updates copyright. 
Please check that everything looks normal and deploy on merge

### CODE REVIEW

File: app/views/desktop/about.ejs [hunk 1]
```diff
@@ -37,6 +37,8 @@ this license in a file with the distribution.
       these meetings.
     </p>
     <p>ACM@UIUC is a registered 501(c)(3) non-for-profit organization.</p>
+    For more on the history of ACM check out our <a href="/about/history">chaper history page</a>
```
Comment (bcongdon) [inline app/views/desktop/about.ejs]: chaper -> chapter

File: app/views/desktop/home.ejs [hunk 2]
```diff
@@ -23,27 +23,24 @@ this license in a file with the distribution.
...
+        <a href="https://hackillinois.org"><button id="history" class="button">Check Out HackIllinois</button></a>
+        <h4>ACM Open House</h4>
+        <p>Held at the beginning of Spring and Fall semesters, this is a great opportunity to learn more about ACM and its various Special Interest Groups (SIGs).</p>
+      </p>
     </div>
     <div id="involvement" class="row">
       <div id="main-left" class="small-12 medium-12 large-4 columns">
         <h3>Join ACM</h3>
-        <p>As an ACM member you get access to our compute cluster, free black and white printing, Beats by ACM, Merch, and many other member perks.</p><a href="/join">
+        <p>As an ACM member you get access to our compute cluster, free black and white printing, Concert, Merch, and many other member perks.</p><a href="/join">
```
Comment (bcongdon) [inline app/views/desktop/home.ejs]: 😉🎧

File: app/views/desktop/home.ejs [hunk 3]
```diff
@@ -23,27 +23,24 @@ this license in a file with the distribution.
...
-      <h4>2014: First ever HackIllinois</h4>
-      <p>In 2014 the first HackIllinois occurred where students from UIUC and across the nation came to Siebel Center to hack for 36 hours. Over the years HackIllinois has been regarded in the hackathon community as an incredibly fun and well-run event, and more recently its focus on introducing students to Open Source has made it unique in the world of hackathons.</p>
-      <h4>2017: ACM@UIUC turns 50 years old</h4>
-      <p>ACM@UIUC turns 50 years old, with about 500 members, 18 SIGs, 2 flagship events and continues to be an open community for those interested in computing.</p>
-      <p></p>
+        <h4>Welcome to ACM@UIUC</h4>
+        <p>The Association for Computing Machinery (ACM) has been around for over 50 years and is a student group consisting of a variety of people with diverse backgrounds who share a common interest in computers.</p>
+        <a href="/about/history"><button id="history" class="button">More About ACM@UIUC's History</button></a>
```
Comment (bcongdon) [inline app/views/desktop/home.ejs]: Stylistic choice, but I'm not sure this button is necessary for the home page. Feels better to just have a link to it under `/about`.

Comment (narendasan) [reply app/views/desktop/home.ejs]: I think it gets hidden in the about page, but it doesnt deserve to be on the nav bar so i though this was a good compromise

**Comment by narendasan** [comment]
there really isnt a reason to remove the happy_birthday command, its just now a way to get confetti on the site now.

File: app/views/desktop/home.ejs (hunk 3 above)
Comment (narendasan) [reply app/views/desktop/home.ejs]: I also added buttons for the other sections to make it standout less

**Comment by bcongdon** [comment]
Alright, looks good then. Thanks for doing the grunt work on this! 🙂

**Comment by narendasan** [comment]
Can you deploy?

**Pull request merged**